### ISSUE TITLE
tasks with full module name fail to run

### ISSUE THREAD

**Issue by lazzarello** [roles-ansible/ansible_role_unbound#1]
```
ERROR! this task 'ansible.builtin.include_tasks' has extra params, which is only allowed in the following modules: group_by, add_host, import_role, ansible.windows.win_command, include_tasks, include, raw, set_fact, import_tasks, command, win_shell, include_role, script, shell, include_vars, win_command, meta, ansible.windows.win_shell

The error appears to be in '/home/lee/.ansible/roles/do1jlr.unbound/tasks/main.yml': line 2, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

---
- name: run optional versionscheck
  ^ here
```

Ansible version info

```
ansible-playbook 2.10.1
  config file = /home/lee/.ansible.cfg
  configured module search path = ['/home/lee/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/lee/.pyenv/versions/3.7.7/lib/python3.7/site-packages/ansible
  executable location = /home/lee/.pyenv/versions/3.7.7/bin/ansible-playbook
  python version = 3.7.7 (default, Aug  1 2020, 23:00:14) [GCC 9.3.0]
```

**Comment by DO1JLR** [comment]
Hello  @lazzarello 

Thank you for opening the issue. Unfortunately I did not manage to reproduce the error.

Ansible has [introduced](https://docs.ansible.com/ansible/devel/porting_guides/porting_guide_2.10.html?highlight=FQCN) the FQCN *(fully-qualified collection name)*  and I started using that in this role as well. This doesn't seem to have really arrived in the ansible version you are using yet.

So if it is an option for you I would suggest to update your ansible. In my experience, the latest version that can be installed via pip has always proven to be quite reliable. With ansible-playbook 2.11 I haven't had any problems using the FQCN yet.

Another alternative would be to go back to the time when the FQCN were not yet in this role: [8ce3f049257c45a5f75513b009f4a83467d5c574](https://github.com/roles-ansible/ansible_role_unbound/tree/8ce3f049257c45a5f75513b009f4a83467d5c574). But I don't think that would be a really good idea.
